config/config.php

```diff
@@ -6,14 +6,14 @@
 $configManager = new ConfigAggregator([
 
     // App packages
-    \Web\ConfigProvider::class,
     \Category\ConfigProvider::class,
     \Article\ConfigProvider::class,
     \Menu\ConfigProvider::class,
     \Admin\ConfigProvider::class,
     \Newsletter\ConfigProvider::class,
     \Page\ConfigProvider::class,
     \Meetup\ConfigProvider::class,
+    \Web\ConfigProvider::class,
```

Comment:
It is the opposite it should be the last one, probably because of FastRoute package. I got this error when i placed Web package as first `Static route "/admin/categories" is shadowed by previously defined variable route "/([^/]+)/([^/]+)" for method "GET"
`